File: listas de exercicio/lista 3/lista 3 casa/questao 10.py
S, comando = str
R = int
if comando == S:
    comando = (str(input("Digite um comando (S, +, -, *, /) para a calculadora:")))
    if comando == "+":
        x = (int(input("Digite o primeiro valor:")))
        y = (int(input("Digite o segundo valor:")))
        R = x + y
        print(f"o resultado é {R}")
    if comando == "-":
        x = (int(input("Digite o primeiro valor:")))
        y = (int(input("Digite o segundo valor:")))
        R = x - y
        print(f"o resultado é {R}")
    if comando == "*":
        x = (int(input("Digite o primeiro valor:")))
        y = (int(input("Digite o segundo valor:")))
        R = x * y
        print(f"o resultado é {R}")
    if comando == "/":
        x = (int(input("Digite o primeiro valor:")))
        y = (int(input("Digite o segundo valor:")))
        R = x / y
        print(f"o resultado é {R}")
    if comando == S:
        break
    

File: listas de exercicio/lista 3/lista 3 casa C/questao 10.c
#include <stdio.h>
#include <stdbool.h>
int main(){
char comando;
int R, x, y;
while (true){
    printf("Digite um comando para a calculadora (+, -, *, /) ou (S, para sair):");
    /*espaço antes da % é para nao repetir a linha print!!!*/
    scanf(" %c", &comando);
    if (comando == '+'){
        printf("Digite o primeiro valor:");
        scanf("%d", &x);
        printf("Digite o segundo valor:");
        scanf("%d", &y);
        R = x + y;
        printf("o resultado é:%d\n", R);
    }
    if (comando == '-'){
        printf("Digite o primeiro valor:");
        scanf("%d", &x);
        printf("Digite o segundo valor:");
        scanf("%d", &y);
        R = x - y;
        printf("o resultado é:%d\n", R);
    }
    if (comando == '*'){
        printf("Digite o primeiro valor:");
        scanf("%d", &x);
        printf("Digite o segundo valor:");
        scanf("%d", &y);
        R = x * y;
        printf("o resultado é:%d\n", R);
    }
    if (comando == '/'){
        printf("Digite o primeiro valor:");
        scanf("%d", &x);
        printf("Digite o segundo valor:");
        scanf("%d", &y);
        R = x / y;
        printf("o resultado é:%d\n", R);
    }
    if (comando == 'S'){
        printf("Calculadora encerrada.");
        break;
    }
  }
}